src/app/app.module.ts

```diff
@@ -44,6 +44,7 @@ import { SliderComponent } from './components/home-page/slider/slider.component'
 import { NewsListItemComponent } from './components/partials/news-list-item/news-list-item.component';
 
 import { environment } from '../environments/environment';
+import { CalPopupComponent } from './components/schedule/calendar/cal-popup/cal-popup.component';
```

Comment:
Don't use such strange short names, right full, please

Reply:
Ohh, I'll fix this in the next commit)